sklearn/ensemble/tests/test_iforest.py

```diff
@@ -322,6 +322,7 @@ def test_iforest_with_uniform_data():
 
     rng = np.random.RandomState(0)
 
+    assert all(np.abs(iforest.decision_function(X)) < 1.0e-15)
```

Comment:
To complete [@albertcthomas's remark](https://github.com/scikit-learn/scikit-learn/pull/19087/files#r664010950).
```suggestion
    assert all(np.abs(iforest.decision_function(X)) < np.finfo(float).eps)
```

Reply:
As above, I had to double it to `2*np.finfo(float).eps`, but hopefully the numerical instability issue will be resolved altogether soon.